.jenkins/macos-build-test.sh

```diff
@@ -18,7 +18,7 @@ export MACOSX_DEPLOYMENT_TARGET=10.9
 export CXX=clang++
 export CC=clang
 # If we run too many parallel jobs, we will OOM
-export MAX_JOBS=2
+export MAX_JOBS=3
```

Comment:
Haha, thanks. We might be able to bump this all the way to 4, actually, since we've got quad core machines. Or 5, to hide IO latency.